qemu/tests/sgx_basic.py

```diff
@@ -0,0 +1,44 @@
+from virttest import error_context
+from virttest.utils_misc import verify_dmesg
+
+from provider.sgx import SGXHostCapability
+from provider.sgx import SGXChecker
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu sgx basic test:
+    1. Check host sgx capability
+    2. Boot sgx VM
+    3. Verify sgx enabled in guest
+    4. Check sgx qmp cmd
+    5. Verify qmp cmd in host and guest sgx capability
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    error_context.context("Start sgx test", test.log.info)
+    timeout = float(params.get("login_timeout", 240))
```

Comment:
I think the statement "timeout = params.get_numeric("login_timeout", 240)" is also OK, for your reference. Thanks.

Reply:
Thanks @XueqiangWei .
Yes, besides get_numeric, we also have some other useful functions supported by utils_params.Params, such as get_boolean/objects, which can be used in case dev, the point is it's better to use what VT supplies if the interfaces/classes/utils have already been supported with VT. cc @zixi-chen